src/ompl/geometric/planners/bitstar/datastructures/CostHelper.h

```diff
@@ -200,23 +212,21 @@ namespace ompl
             /** \brief The true cost of an edge, including constraints.*/
             inline ompl::base::Cost trueEdgeCost(const VertexConstPtrPair &edgePair) const
             {
-                return this->motionCost(edgePair.first->stateConst(), edgePair.second->stateConst());
+                return this->motionCost(edgePair.first->state(), edgePair.second->state());
             };
 
-            /** \brief Combine 3 costs */
-            inline ompl::base::Cost combineCosts(const ompl::base::Cost &a, const ompl::base::Cost &b,
-                                                 const ompl::base::Cost &c) const
+            /** \brief Combine multiple costs. */
+            template <typename... Costs>
+            inline ompl::base::Cost combineCosts(const ompl::base::Cost &cost, Costs... costs) const
```

Comment:
Nice! Shouldn't that be `const Costs... &costs` though?

Reply:
It definitely should, nice catch!